MV.Wildebeest.Core/MV.Wildebeest.Core.iml

```diff
@@ -11,7 +11,7 @@
       <sourceFolder url="file://$MODULE_DIR$/source/core/resources" isTestSource="false" />
       <excludeFolder url="file://$MODULE_DIR$/target" />
     </content>
-    <orderEntry type="inheritedJdk" />
+    <orderEntry type="jdk" jdkName="1.8" jdkType="JavaSDK" />
```

Comment:
@aris-i I'm not sure if I should pull this in - will it break other dev environments?

Reply:
Oops, I didn't noticed that.  I reinstalled my IntelliJ to use community edition instead.  I'll revert the changes on that one.